VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.spec.ts

```diff
@@ -612,9 +613,24 @@ describe('HearingControlsBaseComponent', () => {
     });
 
     it(`canShowScreenShareButton() should return "false" when device is a tablet`, () => {
+        deviceTypeService.isDesktop.and.returnValue(false);
         deviceTypeService.isTablet.and.returnValue(true);
         component.ngOnInit();
-        expect(component.canShowScreenShareButton).toBeFalsy();
+        expect(component.canShowScreenShareButton).toBe(false);
+    });
+
+    it(`canShowScreenShareButton() returns "false" when it is a mobile device`, () => {
+        deviceTypeService.isDesktop.and.returnValue(false);
+        deviceTypeService.isMobile.and.returnValue(true);
+        component.ngOnInit();
+        expect(component.canShowScreenShareButton).toBe(false);
+    });
+
+    it(`canShowScreenShareButton() returns "true" when it is a desktop device`, () => {
+        deviceTypeService.isTablet.and.returnValue(false);
+        deviceTypeService.isDesktop.and.returnValue(true);
```

Comment:
The isMobile and isTablet aren't really relevant here as we only care about isDesktop.